Document bundled https-dns-proxy defaults (#266)
NethServer/nethsecurity-docs: dns_over_http.rst

Installation
============

> The ``https-dns-proxy`` package is included in NethSecurity images, so no

Author: ```suggestion
Since NethSecurity 8.8, the ``https-dns-proxy`` package is included in NethSecurity image, so no
```

> separate installation step is required.

Author: ```suggestion
separate installation step is required.

On NethSecurity 7.7, the package is not included in default NethSecurity image, so you will need to install it manually: ::

  opkg update
  opkg install https-dns-proxy
```

Installation
============

Since NethSecurity 8.8, the ``https-dns-proxy`` package is included in NethSecurity image, so no
separate installation step is required.

> On NethSecurity 7.7, the package is not included in default NethSecurity image, so you will need to install it manually: ::

Author: ```suggestion
On NethSecurity 8.7, the package is not included in default NethSecurity image, so you will need to install it manually: ::
```

Installation
============

Since NethSecurity 8.8, the ``https-dns-proxy`` package is included in NethSecurity image, so no
separate installation step is required.

On NethSecurity 7.7, the package is not included in default NethSecurity image, so you will need to install it manually: ::

  opkg update
  opkg install https-dns-proxy

> 

Author: ```suggestion
```

Image update
------------

The package is included in the image, so it does not need to be reinstalled
after an upgrade.

However, NethSecurity treats ``dnsmasq_config_update='-'`` as the disabled
state. If that value is still set during an image upgrade, the first-boot
defaults script can disable ``https-dns-proxy`` again.

At the moment this is not expected to be a practical problem because

> configuration is supported only from the command line.

Author: ```suggestion
```
